packages/unenv-preset/src/preset.ts

```diff
@@ -64,29 +65,25 @@ export const cloudflare: Preset = {
 		),
```

Comment:
Out of curiosity, why do we create all these identity aliases? It seems a bit pointless?
Shouldn't we only add aliases for things that need to be mapped to polyfills?
(I could understand creating `${p}` -> `node:${p}` perhaps?)

Reply:
I have added this comment:

```
		// `nodeCompatModules` are implemented in workerd.
		// Create alias to override polyfills defined in based environments.
```

Does it help?